code/modules/food_and_drinks/restaurant/generic_venues.dm

```diff
@@ -0,0 +1,115 @@
+/////RESTAURANT/////
+/datum/venue/restaurant
+	name = "restaurant"
+	req_access = ACCESS_KITCHEN
+
+/datum/venue/restaurant/order_food(mob/living/simple_animal/robot_customer/customer_pawn, datum/customer_data/customer_data)
+	var/obj/item/object_to_order = pickweight(customer_data.orderable_objects[type]) //Get what object we are ordering
+
+	object_to_order = new object_to_order()
```

Comment:
Since this order list is static over the round and quite small, wouldn't it be better to build a table for these instead of creating and deleting these every time ?